api/src/main/java/org/apache/cloudstack/api/command/admin/storage/SeedOfficialSystemVMTemplateCmd.java

```diff
@@ -0,0 +1,166 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.cloudstack.api.command.admin.storage;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashSet;
+import java.util.List;
+
+import org.apache.cloudstack.acl.RoleType;
+import org.apache.cloudstack.api.APICommand;
+import org.apache.cloudstack.api.ApiConstants;
+import org.apache.cloudstack.api.BaseCmd;
+import org.apache.cloudstack.api.Parameter;
+import org.apache.cloudstack.api.response.SeedOfficialSystemVMTemplateResponse;
+import org.apache.cloudstack.api.response.ZoneResponse;
+
+import com.cloud.org.Cluster;
+import com.cloud.user.Account;
+import com.cloud.utils.Pair;
+import com.cloud.utils.exception.CloudRuntimeException;
+import com.cloud.utils.http.downloader.HttpDirectTemplateDownloader;
+import com.cloud.utils.script.Script;
+
+@APICommand(name = "seedOfficialSystemVMTemplate", description = "Copies an official system vm template into secondary storage",
+        responseObject = ZoneResponse.class, requestHasSensitiveInfo = false, responseHasSensitiveInfo = false, since = "4.13",
+        authorized = {RoleType.Admin})
+public class SeedOfficialSystemVMTemplateCmd extends BaseCmd {
+
+    public Long getId() {
+        return id;
+    }
+
+    @Parameter(name = ApiConstants.ID, type = CommandType.UUID, entityType = ZoneResponse.class, required = true, description = "the ID of the zone")
```

Comment:
Reformat the API implementation, see other/existing APIs how they're formatted. 
We would for example prefer the params to be declared before the accessors (getters/setters); and then the execute() etc. methods.